src/components/sections/skills.tsx

```diff
@@ -0,0 +1,33 @@
+import { skills } from "@/constants"
+import { useMediaQuery } from "@/hooks"
+import React from "react"
+
+interface Props {}
+
+export const Skills: React.FC<Props> = () => {
+  const isDesktop = useMediaQuery("(min-width: 1025px)")
```

Comment:
same as my message above, this can surely be made without using hooks or any js just css, so if you want give it a try as a practice: no `span-` just flex stuff